src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawLineAdapter.java

```diff
@@ -26,12 +27,15 @@ public void operateTo(int x, int y) {
 		ILine line = LineFactory.getBasicLine();
 		line.setStartCoordinates(this.startX, this.startY);
 		line.setEndCoordinates(x, y);
+		setPosition(x, y);
+
+		DrawPanelController drawPanelControll = DrawerFeature.getDrawerController();
+		drawPanelControll.drawLine(line);
```

Comment:
Constructor injection should be used (using a parameter). This is unnecessary direct coupling with DrawerFeature class.